files/galaxy/tpv/tools.yml

```diff
@@ -106,6 +106,37 @@ tools:
     cores: 10
     mem: 32
     rules:
+        # Note: As of today, there is no `tool_version_eq` function in TPV.
```

Comment:
FYI: AFAIK `eq` was never implemented. The last time when I looked into the TPV helpers codebase I did not see that and nor did the [PR](https://github.com/galaxyproject/total-perspective-vortex/pull/91/files) created that added the rest of the comparison.

Reply:
I have created an [PR](https://github.com/galaxyproject/total-perspective-vortex/pull/98) for this in the upstream